### ISSUE TITLE
How to understand batch_size and batch in Grid_pool

### ISSUE THREAD

**Issue by Stronger-Huang** [Pointcept/Pointcept#225]
Hello, thanks for your contribution of PTv2.
You set batch_size=12 in the cfg file, but train_loader brings in each time is two-dimensional, for example: coord: [num_points, 3]. Is this data one of batch?(12 point clouds a time ?)

In grid_pool function : what the 'batch' means in this function? The batch converted by offset is [0,0,...,1,1,...,5,5]. And it changes with batch_size.

So What the difference between " batch_size " and "batch" in Grid_pool?

I would be grateful if you could answer this question!!

**Comment by Gofinge** [comment]
Readme here (https://github.com/Pointcept/Pointcept?tab=readme-ov-file#offset) will be helpful. For real-world data, as each point cloud contains a different number of points, we have to merge the batch (b) and num_point (n) into one dimension.

**Comment by Stronger-Huang** [comment]
Thank you for your reply!!

The input of PTv2 contains 720,000 points, so is this the result of adding up 12 batches? Then why is the batch from 0 to 5 (6 batches?), and how many points are there in s3dis?

**Comment by Gofinge** [comment]
> The input of PTv2 contains 720,000 points, so is this the result of adding up 12 batches?

Yes.

> Then why is the batch from 0 to 5 (6 batches?)

Sorry, I can't understand the question.

> How many points are there in s3dis?

You can check the process data by printing the number.

**Comment by Stronger-Huang** [comment]
Thank you for your reply!!

> Then why is the batch from 0 to 5 (6 batches?)

I mean I print variable batch in Grid Pool: [0,0,0,...,1,1,1,...5,5,5], it's separator in 12 batches of Point Cloud.
But why only 0 to 5 (seems 6 batches)

**Comment by Stronger-Huang** [comment]
Thank you for your reply!!
how much can this mixing method improve the accuracy?
